src/utils/getURL.js

```diff
@@ -0,0 +1,10 @@
+function getURL(path, config) {
```

Comment:
you can also make config a default param {}, so you don't have to pass an empty object whenever you fetch